evennia/web/api/permissions.py

```diff
@@ -0,0 +1,57 @@
+from rest_framework import permissions
+
+
+class EvenniaPermission(permissions.BasePermission):
+    """
+    A Django Rest Framework permission class that allows us to use
+    Evennia's permission structure. Based on the action in a given
+    view, we'll check a corresponding Evennia access/lock check.
+    """
+    # subclass this to change these permissions
+    MINIMUM_LIST_PERMISSION = "builder"
```

Comment:
I wonder if one could not link this into the settings file's permission levels instead somehow.